### DISCUSSION TITLE
RuntimeError: Magick: delegate failed `'curl' -s -k -L -o '%u.dat'

### DISCUSSION THREAD

**Discussion by jackthenewbie** [ImageMagick/ImageMagick#6241] [Help] [answered]
### ImageMagick version

7

### Operating system, version and so on

ubuntu docker latest image

### Description

``RuntimeError: Magick: delegate failed `'curl' -s -k -L -o '%u.dat' 'https:%M'' @ error/delegate.c/InvokeDelegate/1924``

I'm using Magick++ in my project, where I pass direct url (from backblaze b2 bucket) to ``Magick::Image``, it was working fine on debian 11. But when I use ubuntu latest image from docker hub, I got that error. I had checkout #230 , but not sure how to debug it on cpp. 
This is how I build ImageMagick

```docker
FROM ubuntu
ENV DEBIAN_FRONTEND noninteractive
WORKDIR /home/pifordecoyer/code/cpp/GenshinCard
COPY dependencies .
RUN apt-get update -y && \
    apt-get install -y python3.10 python3-pip git-all build-essential pkg-config `cat dependencies`
WORKDIR /app/ImageMagick
RUN git clone https://github.com/ImageMagick/ImageMagick.git /app/ImageMagick && \
    ./configure CXX=g++ && \
    make && \
    make install
RUN git clone https://github.com/libcpr/cpr.git /app/cpr
WORKDIR /app/cpr/build
RUN cmake /app/cpr -DCPR_BUILD_TESTS=OFF && \
    make install
```

**Comment by dlemstra** [comment, marked as answer]
It looks like you forgot to install curl?

**Reply by jackthenewbie** [reply]
Totally forgot to install curl, thanks:)
